devito/symbolics/extended_sympy.py

```diff
@@ -202,6 +202,52 @@ def base(self):
     __reduce_ex__ = Pickable.__reduce_ex__
 
 
+class ExternalFunctionCall(sympy.Expr, Pickable):
```

Comment:
could you add a pickle test next to those for the other sympy objects implemented in this file? you'll see it's just two lines